tests/llmcompressor/pipelines/test_cache.py

```diff
@@ -28,6 +28,7 @@ def sample_cache(sample_dataloader):
     )
 
 
+@pytest.mark.unit
```

Comment:
A side question: what does this marker do? Do we run a subset of tests based on the marker?

Reply:
I don’t think we use the marker now, but it allows us in the future to skip the unit tests if we thunk they’re taking too long